.circleci/config.yml

```diff
@@ -117,13 +117,19 @@ commands:
       - run:
           name: Run cucumber scenarios
           command: cd integration_tests && mkdir -p cucumber_output && node_modules/.bin/cucumber-js --tags "not @long-running and not @broken and not @flaky" --format json:cucumber_output/tests.cucumber
```

Comment:
```suggestion
          command: cd integration_tests && mkdir -p temp/upload && mkdir -p cucumber_output && node_modules/.bin/cucumber-js --tags "not @long-running and not @broken and not @flaky" --format json:cucumber_output/tests.cucumber
```

Reply:
I don't think this is needed

Reply:
I basically propose we zip the log files to `integration_tests/temp/upload` and specify that as the `store_artifacts` path. The rest of my comments are in line with that.